tensorflow/python/training/learning_rate_decay_test.py

```diff
@@ -73,5 +75,67 @@ def testVariables(self):
       self.assertAllClose(decayed_lr.eval(), expected, 1e-6)
 
 
+class ExponentialDecayTest(test_util.TensorFlowTestCase):
+
+  def testDecay(self):
+    initial_lr = 0.1
+    k = 10
+    decay_rate = 0.96
+    with self.test_session():
+      for step in range(0, k+1):
+        lr = learning_rate_decay.natural_exp_decay(
```

Comment:
these tests are creating 10 copies of the graph -- can you rewrite the test so that it creates the graph once and passes the proper parameters as inputs to the function? 


You can see how it's done for the exponential case above: all the ops are created first rather than creating them in a loop.